widget/embedded/src/store/slices/data.ts

```diff
@@ -96,12 +99,38 @@ export const createDataSlice: StateCreator<
       type: options.type,
     });
 
+    const includedTokens = new Set();
+    const excludedTokens = new Set();
+
+    if (supportedTokensConfig) {
+      if (Array.isArray(supportedTokensConfig)) {
+        supportedTokensConfig.forEach((token) =>
+          includedTokens.add(createTokenHash(token))
+        );
+      } else if (!Array.isArray(supportedTokensConfig)) {
```

Comment:
Do we need this condition?
`Array.isArray(supportedTokensConfig)` in else conditions is always `false`